mava/systems/ff_ippo.py

```diff
@@ -522,7 +522,33 @@ def run_experiment(_config: Dict) -> None:
     # Calculate total timesteps.
     n_devices = len(jax.devices())
     config["arch"]["devices"] = jax.devices()
+    if config["system"]["total_timesteps"] is None:
+        config["system"]["total_timesteps"] = (
+            n_devices
+            * config["system"]["num_updates"]
+            * config["system"]["rollout_length"]
+            * config["system"]["update_batch_size"]
+            * config["arch"]["num_envs"]
+        )
+    else:
+        config["system"]["num_updates"] = (
+            config["system"]["total_timesteps"]
+            // config["system"]["rollout_length"]
+            // config["system"]["update_batch_size"]
+            // config["arch"]["num_envs"]
+            // n_devices
+        )
+        print(
+            f"{Fore.RED}{Style.BRIGHT} Updated number of updates"
+            + f"to {config['system']['num_updates']}: If you want to train"
+            + " for a specific number of updates, please set total_timesteps to None"
+            + f"{Style.RESET_ALL}"
+        )
```

Comment:
Since we're going to have to do this in all files can you please put this in a utils method, maybe in a new file e.g `utils/total_timestep_checker.py`

Reply:
Nice idea!!